lib/bundle/cask_dumper.rb

```diff
@@ -13,15 +13,33 @@ def casks
 
       require "cask/caskroom"
 
-      @casks ||= Cask::Caskroom.casks.map(&:to_s)
-      @casks.map { |cask| cask.chomp " (!)" }
-            .uniq
+      @casks ||= Cask::Caskroom.casks.map(&:to_s).uniq
+    end
+
+    def casks_config
+      return {} unless Bundle.cask_installed?
+
+      @casks_config ||= Hash[Cask::Caskroom.casks.map(&:to_s).zip(
```

Comment:
I think it'd be worth caching the result of `Cask::Caskroom.casks` into a new method so we never call it multiple times.

Reply:
I'd also separate out the `Hash[` here; it reads a bit weird.

Reply:
By "separate out", I take it you mean add a line break.